mirage/config.js

```diff
@@ -185,8 +185,9 @@ export default function() {
 
   this.post('/password/reset', (_schema, request) => {
     let params = request.requestBody;
-    let password = params.split('&')[0].split('=')[1];
-    let passwordConfirmation = params.split('&')[1].split('=')[1];
+    let [first, second] = params.split('&');
+    let password = first.split('=')[1];
+    let passwordConfirmation = second.split('=')[1];
```

Comment:
There should be a `this._getAttrsForFormRequest()` which returns a POJO containing the form data, doing this job for you. It's undocumented, but it is supported by mirage.

However, you are using an arrow function, so you do not have access to `this`.

I believe something like
```Javascript
this.post('/password/reset', function() {
  let { password, passwordConfirmation } = this._getAttrsForFormRequest();

  // continue with the rest of your code here
));
```
ought to do the trick.